Adding PowerShell & Graph permission information as well as link to Graph documentation (#10382)
MicrosoftDocs/msteams-docs: msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

<b> After March 14th, 2024 </b>

> Before you can manage your Team and Chat RSC settings with PowerShell, you need to connect PowerShell to your tenant using Microsoft Graph. You can learn more about managing Microsoft Graph settings with PowerShell [here](/powershell/microsoftgraph/get-started?view=graph-powershell-1.0).

Reviewer: ```suggestion
Before you manage your team and chat RSC settings with PowerShell, you need to connect PowerShell to your tenant using Microsoft Graph. For more information on managing Microsoft Graph settings with PowerShell, see [get started with the Microsoft Graph PowerShell SDK](/powershell/microsoftgraph/get-started?view=graph-powershell-1.0).
```

> You can use the `Connect-MgGraph` cmdlet to do this. You will need to connect with the following permissions. 

Reviewer: ```suggestion
You can use the `Connect-MgGraph` cmdlet and connect with the following permissions:
```
